netfile/management/commands/downloadnetfilerawdata.py

```diff
@@ -105,9 +106,11 @@ def handle(self, *args, **options):
 
     def combine(self):
         headers_written = False
+        fname_pattern = re.compile('netfile_\d{4}_(.+)_cal201')
+        #                                          ^^ Agency shortcut in filename 
         with file(self.combined_csv_path, 'w') as combined_csv:
             for path in glob.glob(os.path.join(self.data_dir, 'netfile_*_*_cal201_export.csv')):
-                agency_shortcut = path.split('_')[2]
```

Comment:
I think you could accomplish the same thing without a regular expression, just change this line to:

    agency_shortcut = os.path.basename(path).split('_')[2]